include/antler/project/dependency.hpp

```diff
@@ -33,16 +33,15 @@ class dependency {
    /// @param tag  This is either the github repo tag or commit hash, commit hash being preferable. This may be empty if rel is populated or loc points to an archive.
    /// @param rel  This is a github repo version. Using a commit hash tag is preferable. This may be empty.
    /// @param hash  If loc points to an archive, this should be populated with the sha256 hash.
-   inline dependency(const std::string& loc, const std::string& name = "", const std::string& tag="",
-                     const std::string& rel="", const std::string& hash="") {
+   inline dependency(const std::string& loc, const std::string& name = "", const std::string& tag = "", const std::string& rel = "", const std::string& hash = "") {
```

Comment:
It seems Google style by default puts arguments / members in the same line. This style would have to be used with `ColumnLimit`.